Examples/MAX32655/Unity_Test/project.mk

```diff
@@ -0,0 +1,43 @@
+# This file can be used to set build configuration
+# variables.  These variables are defined in a file called 
+# "Makefile" that is located next to this one.
+
+# For instructions on how to use this system, see
+# https://analogdevicesinc.github.io/msdk/USERGUIDE/#build-system
+
+# **********************************************************
+
+# Add your config here!
+LIB_UNITY=1
+
+
+TEST_CC=gcc
+TEST_CFLAGS=-std=c89
+TEST_CFLAGS += -Wall
+TEST_CFLAGS += -Wextra
+TEST_CFLAGS += -Wpointer-arith
+TEST_CFLAGS += -Wcast-align
+TEST_CFLAGS += -Wwrite-strings
+TEST_CFLAGS += -Wswitch-default
+TEST_CFLAGS += -Wunreachable-code
+TEST_CFLAGS += -Winit-self
+TEST_CFLAGS += -Wmissing-field-initializers
+TEST_CFLAGS += -Wno-unknown-pragmas
+TEST_CFLAGS += -Wstrict-prototypes
+TEST_CFLAGS += -Wundef
+TEST_CFLAGS += -Wold-style-definition
```

Comment:
Do we strictly need this?  Related to the question about the custom target recipe

Reply:
No. Inside the framework is an examples folder, which I originally pulled from. This was just what came with the example. Left it there fro now, but it should use as much of the same flags the normal build uses.